commercetools.Sdk/commercetools.Sdk.Serialization/JsonConverters/SerializeAsDateConverter.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using commercetools.Sdk.Domain;
+
+namespace commercetools.Sdk.Serialization
+{
+    internal class SerializeAsDateConverter : JsonConverterBase
+    {
+        private readonly IsoDateTimeConverter dateOnlyConverter = new IsoDateTimeConverter
+        {
+            DateTimeFormat = "yyyy-MM-dd"
+        };
+        private string DateOnlyProp { get; set; }  
+        public override List<SerializerType> SerializerTypes => new List<SerializerType>() { SerializerType.Serialization };
+
+        public override bool CanConvert(Type objectType)
+        {
+            var hasDateOnlyProps= objectType.IsDefined(typeof(SerializeAsDateOnlyAttribute));
+            if (hasDateOnlyProps)
+            {
+                var attribute = objectType.GetCustomAttribute<SerializeAsDateOnlyAttribute>();
+                DateOnlyProp = attribute.PropertyName;
```

Comment:
it may be possible that there is more than one date only prop.
And you may have to check here too that a SerializeAsTimeOnlyAttribute has been defined